src/typechecker/environment.lisp

```diff
@@ -513,9 +513,9 @@
   '(satisfies struct-field-list-p))
 
 (defstruct struct-entry
-  (name             (util:required 'name)      :type symbol            :read-only t)
```

Comment:
Maybe move indentation fixes to a separate PR

Reply:
done